File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        int[] result = new int[nums.length];

        for (int i = 0; i < nums.length; i++) {
            int product = 1;
            for (int j = 0; j < nums.length; j++) {
                if (j != i && nums[j] == 0) {
                    product = 0;
                    break;
                }
                else if (j != i) product *= nums[j];
            }
            result[i] = product;
            product = 1;
        }
        return result;
    }
}  

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        if not nums:
            return 0
        
        nums.sort()

        max = 0
        length = 0
        for i in range(len(nums) - 1):
            if nums[i] == nums[i+1]:
                i = i + 1
            elif nums[i+1] - nums[i] == 1:
                length = length + 1
                if length > max:
                    max = length
            else:
                length = 0
        return max + 1